src/components/Home/JuicecrowdBanner.tsx

```diff
@@ -0,0 +1,19 @@
+import { Trans } from '@lingui/macro'
+import Banner from 'components/Banner'
+import ExternalLink from 'components/ExternalLink'
+
+export function JuicecrowdBanner() {
+  return (
+    <Banner
+      body={
+        <Trans>
+          Juicecrowd submissions are now open.{' '}
+          <strong className="font-semibold">3 ETH</strong> prize pool.{' '}
+          <ExternalLink href="https://juicecrowd.gg">Learn more</ExternalLink>
```

Comment:
Wrap Learn More with a Trans

Reply:
Could be wrong but I don't think it needs it. Lingui handles inner elements like this, can still be translated:
<img width="746" alt="Screen Shot 2023-10-21 at 3 14 32 pm" src="https://github.com/jbx-protocol/juice-interface/assets/96150256/7fd49c21-3f86-470c-b0c3-167dd619a3e3">